locations/spiders/bobs_discount_furniture_us.py

```
@@ -0,0 +1,64 @@
+from scrapy.spiders import XMLFeedSpider
+
+from locations.categories import Categories, apply_category
+from locations.hours import OpeningHours
+from locations.items import Feature
+
+
+class BobsDiscountFurnitureUSSpider(XMLFeedSpider):
+    name = "bobs_discount_furniture_us"
+    item_attributes = {"brand": "Bob's Discount Furniture", "brand_wikidata": "Q4931582"}
+    allowed_domains = ["www.mybobs.com"]
+    start_urls = ["https://api.mybobs.com/occ/v2/bobsspa/stores?pageSize=1000&fields=FULL"]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)